.github/workflows/ruby.yml

```diff
@@ -0,0 +1,26 @@
+name: Ruby
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6.x
+    - name: Build and run Rubocop
+      run: |
+        gem install bundler
+        bundle install --jobs 4 --retry 3
+        bundle exec rubocop --parallel
+    # TODO:
+    # - name: Build and run tests
+    #   run: |
+    #     gem install bundler
+    #     bundle install --jobs 4 --retry 3
+    #     bundle exec rake
```

Comment:
Eventually this can/should be used to run unit tests that don't have a dependency on the sandbox